_currencies.md

```diff
@@ -87,6 +87,12 @@ curl https://api.uphold.com/v0/assets \
   -H "Authorization: Bearer <token>"
 ```
 
+> Example of filtering the list to show only assets of specific `type`s:
```

Comment:
Is "`type`s" intended?

Reply:
Yes, the "type" part of the word is highlighted because it's the specific keyword that should be used in the URL.